pandas/tests/resample/test_datetime_index.py

```diff
@@ -412,70 +412,6 @@ def test_resample_frame_basic():
     df.resample("W-WED", kind="period").mean()
 
 
-@pytest.mark.parametrize(
-    "loffset", [timedelta(minutes=1), "1min", Minute(1), np.timedelta64(1, "m")]
-)
```

Comment:
same as above

Reply:
They have been moved into `test_deprecated.py`.
When you told me to remove some deprecated tests I moved them instead into `test_deprecated.py`: https://github.com/pandas-dev/pandas/pull/31809#discussion_r379711472